src/svtk/svtk/pesr/pe_test.py

```diff
@@ -100,7 +100,7 @@ def _get_coords(pos, strand):
         i = 0
         for pair in pairs:
             pair = _DiscPair(*pair)
-            if(i > 1000000):
+            if (i > 1000000):
```

Comment:
Parenthesis have no effect
```suggestion
            if i > 1000000:
```